db/migrate/20241204111437_cleanup_invalid_partner_profiles.rb

```diff
@@ -0,0 +1,61 @@
+class CleanupInvalidPartnerProfiles < ActiveRecord::Migration[7.1]
+  def up
+    # ActiveRecord::Base.logger = Logger.new(STDOUT)
+    invalid_profiles = Partners::Profile.all.reject(&:valid?)
+
+    return if !invalid_profiles
```

Comment:
nit: Since `invalid_profiles` will always be an array, it will always be truthy so this guard statement isn't doing much atm. Maybe Rails helpers like `.present?` or `.blank?` would be useful here?

Not a big deal though because if `invalid_profiles` is an empty array the block below won't execute.

Reply:
Thanks for the nit!